build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -132,16 +127,20 @@ module.exports = function ({types: t}) {
           }
         }
 
-        if (conversions === 0) {
+        newQuasis = newQuasis.filter(Boolean);
+        if (newQuasis.length === 1) {
+          // When the remaining number of quasis is one.
+          // Replace the TemplateLiteral with a StringLiteral.
+          // `foo` => 'foo'
+          path.replaceWith(t.stringLiteral(newQuasis[0].value.raw));
```

Comment:
This needs to be the `cooked` value to ensure escaping is preserved. See example at https://astexplorer.net/#/gist/ee7d0673c706c9176890bb7e61d1a1f7/38de7eee4190069c3838a0121a722b75dede21ac